### PR TITLE
[WIP] Added bundle autoloading for Contao Manager

### PR DESCRIPTION
**Pull request by aschempp**
see https://github.com/contao/manager-bundle

### CODE REVIEW

**Comment by leofeyer** [comment]
I would prefer the files to be in a separate namespace.

**Comment by aschempp** [comment]
done

File: src/Manager/ContaoManagerPlugin.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CalendarBundle\Manager;
```
Comment (leofeyer) [inline src/Manager/ContaoManagerPlugin.php]: I thought you (or @Toflar?) had suggested to use `ContaoManager` as namespace?

Comment (aschempp) [inline src/Manager/ContaoManagerPlugin.php]: Hardly feels right to use **Contao**\CalendarBundle\**Contao**Manager

Comment (leofeyer) [inline src/Manager/ContaoManagerPlugin.php]: What about `Isotope\CoreBundle\ContaoManager`?

Comment (aschempp) [inline src/Manager/ContaoManagerPlugin.php]: ¯_(ツ)_/¯

**Comment by leofeyer** [comment]
Merged in 69fbeeea1cad1852a9c669a371e4ebf87c7d3002.

**Pull request merged**